substrate/frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -81,6 +83,58 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		.any(|c| matches!(c.composite_keyword, CompositeKeyword::SlashReason(_)))
 		.then_some(quote::quote!(SlashReason,));
 
+	let call_part_v2 = def.call.as_ref().map(|_| quote::quote!(+ Call));
+
+	let task_part_v2 = def.task_enum.as_ref().map(|_| quote::quote!(+ Task));
```

Comment:
It would probably be cleaner to have a `v2/` folder in frame-support-procedural, but i imagine it would be a huge refactor 😅

Reply:
Yeah, I was hoping to reduce the diff but happy to change if needed.

Reply:
Nah its fine for now.  
Just in the future we should focus on clarity, not smaller diffs 😛